src/plugins/intel_cpu/src/exec_network.cpp

```diff
@@ -126,6 +128,18 @@ ExecNetwork::ExecNetwork(const InferenceEngine::CNNNetwork &network,
         do {
             for (auto&& task : tasks) {
                 task = [this] {
+                    // SSE runtime check is needed for some ATOM machine, which is x86-64 but w/o SSE
```

Comment:
1. I would say it is better to wrap this code block into `ExecNetwork::GetGraph();` itself.
2. We need to make sure StreamExecutor will execute the task not in main thread. From my understanding this is not guaranteed in general (OMP, SEQ modes?). @wangleis CAn you help us find out most reliable way to unsure separate thread will be created for the task execution?

Reply:
For question 2: **OMP mode**, I just verified: OMP and TBB are same, `ExecNetwork::GetGraph()` is executed in different thread. @wangleis , @dmitry-gorokhov

Reply:
For question 1: I can't wrap them into `GetGraph`, because `GetGraph` is const function, and we need to update value "`_cfg.DAZOn`".
If we wrap them into `GetGraph`, we will get **error**:
```
openvino/src/plugins/intel_cpu/src/exec_network.cpp: In member function ‘ov::intel_cpu::ExecNetwork::GraphGuard::Lock ov::intel_cpu::ExecNetwork::GetGraph() const’:
openvino/src/plugins/intel_cpu/src/exec_network.cpp:170:30: error: assignment of member ‘ov::intel_cpu::Config::DAZOn’ in read-only object
  170 |             this->_cfg.DAZOn = denormals_as_zero(true);
```